targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_MODULE_UBLOX_ODIN_W2/sdk/ublox-odin-w2-drivers/UbloxWiFiConfigInterface.h

```diff
@@ -0,0 +1,52 @@
+/* ODIN-W2 implementation of WiFi Config Interface
+ * Copyright (c) 2016 u-blox Malm� AB
```

Comment:
year should be 2019

Add please SPDX identifier as well

Reply:
> also here, SPDX identifier

already there LICENSE-2.0 ?

Reply:
`SPDX-License-Identifier: Apache-2.0` is needed, as in other files